src/realm/sync/noinst/client_impl_base.cpp

```diff
@@ -1869,17 +1870,18 @@ void Session::send_refresh_message()
     REALM_ASSERT(!m_unbind_message_sent);
     REALM_ASSERT(!m_access_token_sent);
 
-    const std::string& signed_access_token = get_signed_access_token();
-    std::size_t signed_access_token_size = signed_access_token.size();
+    // Discard the token since it's ignored by the server.
+    std::string empty_access_token{};
+    std::size_t signed_access_token_size = empty_access_token.size();
```

Comment:
We don't actually send REFRESH messages anymore - and haven't for years. Since Realm was acquired and we switched to baas (i.e. 2+ years ago), we refresh the tokens out-of-band here https://github.com/realm/realm-core/blob/b8adc4c397e7fbae229660befcaeac96b156694a/src/realm/object-store/sync/app.cpp#L940. Let's just rip this out totally. We can rip out all the code for sending refresh messages. Since the server has always just silently ignored these messages (the implementation is literally an empty function) I don't even think this is a breaking change.